pennylane/templates/layers/simplified_two_design.py

```diff
@@ -0,0 +1,192 @@
+        weights (array[float]): array of rotation angles for the layers, shape ``(L, d, 2)``
+        wires (Sequence[int] or int): qubit indices that the template acts on
+
+    Raises:
+        ValueError: if inputs do not have the correct format
+
+    .. UsageDetails::
+
+        The template is used inside a qnode:
+
+        .. code-block:: python
+
+            import pennylane as qml
+            from pennylane.templates import SimplifiedTwoDesign
+            from math import pi
+
+            n_wires = 3
+            dev = qml.device('default.qubit', wires=n_wires)
+
+            @qml.qnode(dev)
+            def circuit(initial_block, weights):
+                SimplifiedTwoDesign(initial_layer=initial_block, weights=weights, wires=range(n_wires))
+                return [qml.expval(qml.PauliZ(wires=i)) for i in range(n_wires)]
+
+            initial_block = [pi, pi, pi]
+            weights_layer1 = [[0., pi],
+                              [0., pi]]
+            weights_layer2 = [[pi, 0.],
+                              [pi, 0.]]
+            weights = [weights_layer1, weights_layer2]
+
+            >>> circuit(initial_block, weights)
+            [1., -1., 1.]
+
+        **Parameter initialization function**
+
+        The :mod:`~pennylane.init` module contains four parameter initialization functions:
+
+        * ``simplified_two_design_initial_layer_normal``
+        * ``simplified_two_design_initial_layer_uniform``
+        * ``simplified_two_design_weights_normal``.
+        * ``simplified_two_design_weights_uniform``.
+
+        They can be used as follows:
+
+        .. code-block:: python
+
+            from pennylane.init import (simplified_two_design_initial_layer_normal,
+                                        simplified_two_design_weights_normal)
+
+            n_layers = 4
+            initial_layer = simplified_two_design_initial_layer_normal(n_wires)
+            weights = simplified_two_design_weights_normal(n_layers, n_wires)
+
+            >>> circuit(initial_layer, weights)
+
+    """
+
+    #############
+    # Input checks
+
+    wires = _check_wires(wires)
+
+    repeat = _check_number_of_layers([weights])
+
+    _check_type(
+        initial_layer,
+        [list, np.ndarray],
+        msg="'initial_layer' must be of type list or np.ndarray; got type {}".format(
+            type(initial_layer)
+        ),
+    )
+    _check_type(
+        weights,
+        [list, np.ndarray],
+        msg="'weights' must be of type list or np.ndarray; got type {}".format(type(weights)),
+    )
+
+    expected_shape_initial = (len(wires),)
+    _check_shape(
+        initial_layer,
+        expected_shape_initial,
+        msg="'initial_layer' must be of shape {}; got {}"
+        "".format(expected_shape_initial, _get_shape(initial_layer)),
+    )
+
+    if len(wires) in [0, 1]:
+        expected_shape_weights = (0,)
+    else:
+        expected_shape_weights = (repeat, len(wires) // 2 + (len(wires) - 1) // 2, 2)
```

Comment:
Same as above: `len(wires) // 2 + (len(wires) - 1) // 2` is equal to `len(wires) - 1`.